### PR TITLE
EXTJS-26323 | Add a Kitchensink example in ExtReact examples section

### PR DESCRIPTION
**Pull request by simran-git**
https://sencha.jira.com/browse/EXTJS-26323
Added a kitchen sink example for split button in Ext-React

### CODE REVIEW

File: packages/reactor-kitchensink/src/examples/Button/SplitButton.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+                        </Container>
+                    </Container>
+                </Panel>
+            </Container>
+        )
+    }
+}
+
+const menuProps ={
+                menu: [{
```
Comment (dbuhrman) [inline packages/reactor-kitchensink/src/examples/Button/SplitButton.js]: Please fix indentation here.

File: packages/reactor-kitchensink/src/examples/Button/SplitButton.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        return (
+            <Container padding="10">
+                <Container 
+                    layout={{ type: 'hbox', pack: Ext.os.is.Phone ? 'center' : 'left'}} 
+                    margin="0 0 10 0" 
+                    defaults={{ margin: "0 10 0 0" }}
+                    width={Ext.isIE && 550}
+                >
+            
+            <Button ui="action raised" text="Style">
```
Comment (dbuhrman) [inline packages/reactor-kitchensink/src/examples/Button/SplitButton.js]: Indentation here.

File: packages/reactor-kitchensink/src/examples/Button/SplitButton.js (hunk 1 above)
Comment (simran-git) [reply packages/reactor-kitchensink/src/examples/Button/SplitButton.js]: Fixed the indentation issues.

**Comment by vaibhavjain3** [comment]
@aphex Please review the PR

**Pull request merged**